sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -561,16 +561,9 @@ case class TruncateTableCommand(
         }
       }
     }
-    // After deleting the data, invalidate the table to make sure we don't keep around a stale
-    // file relation in the metastore cache.
-    spark.sessionState.refreshTable(tableName.unquotedString)
```

Comment:
can we update the doc of these 2 `refreshTable` and make it clear when we should use which?

Reply:
Let me document them separately from this PR.